flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/api/BatchTableEnvironment.scala

```diff
@@ -121,7 +123,9 @@ class BatchTableEnvironment(
   override private[flink] def translateToExecNodeDag(rels: Seq[RelNode]): Seq[ExecNode[_, _]] = {
     val nodeDag = super.translateToExecNodeDag(rels)
     // breakup deadlock
-    new DeadlockBreakupProcessor().process(nodeDag)
+    val postNodeDag = new DeadlockBreakupProcessor().process(nodeDag)
```

Comment:
Looks `DeadlockBreakupProcessor` is a `DAGProcessor` too. Just let it implement `DAGProcessor` too?